VirtoCommerce.Storefront.Model/Security/MemberIdIdentityResult.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Identity;
+
+namespace VirtoCommerce.Storefront.Model.Security
+{
+    public class MemberIdIdentityResult : IdentityResult
```

Comment:
I suggest defining a new type of `UserActionIdentityResult` instead of deriving from `IdentityResult` that will contain all properties from `IdentityResult` to preserving backward compatibility plus in addition with the new  `User` property.